cpp/oneapi/dal/backend/primitives/rnd/rnd_seq_dpc.cpp

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/backend/primitives/rnd/rnd_seq.hpp"
+
+namespace oneapi::dal::backend::primitives {
+
+#ifdef ONEDAL_DATA_PARALLEL
+
+#define INSTANTIATE(F) template class rnd_seq<F>;
+
+INSTANTIATE(float)
+INSTANTIATE(double)
```

Comment:
Looks like instantiation is not needed, we have all functionality implemented in header.

Reply:
Absence of those lines results in undefined references.